app/views/admin/resources/_update_actions.html.haml

```diff
@@ -0,0 +1,10 @@
+.btn-group
+	%button.btn.btn-default.quantity_left.disabled
+		%span{ id: "resource_left_#{resource.id}" }
+			=quantity_left_of(resource)
+	%button{ class: "btn btn-default #{ 'disabled' if resource.used == resource.quantity }" }
+		=link_to admin_conference_resource_path(conference.short_title, resource,resource:{ used: resource.used + 1 }), method: :put, remote: true do
+			%span.fa.fa-minus.fa-2x.update_action
+	%button{ class: "btn btn-default #{ 'disabled' if resource.used == 0  }" }
+		=link_to admin_conference_resource_path(conference.short_title, resource,resource:{ used: resource.used-1 }), method: :put, remote: true do
+			%span.fa.fa-plus.fa-2x.update_action
```

Comment:
Please add empty line at the end of the file, it will make this red indication go away.